pkg/config/config.go

```diff
@@ -155,8 +157,9 @@ type ClusterNetwork struct {
 }
 
 type HarvesterConfig struct {
-	ServerURL string `json:"serverUrl,omitempty"`
-	Token     string `json:"token,omitempty"`
+	SchemeVersion uint32 `json:"schemeVersion,omitempty"`
```

Comment:
Can u specify what the `SchemeVersion` is used for?

Reply:
Currently, we only logged the version but do nothing about this.
It's reserved for the future.
if we want to do some migration in the future.

Reply:
Please add an annotation.